cargo-dist/Cargo.toml

```diff
@@ -48,6 +44,7 @@ parse-changelog = { version = "0.5.3", default-features = false }
 newline-converter = "0.2.2"
 axoproject = { version = "0.3.0", default-features = false, features = ["cargo-projects"] }
 dialoguer = "0.10.4"
+axoasset = "0.2.0"
```

Comment:
In my ideal world we have some pretty fine-grained feature flags on axoasset to toggle the various compression formats on, to avoid all our projects getting bloated with unused dependencies.

Reply:
That sounds like a good idea!